next-i18next.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     // so switching to other languages is temporarily disabled.
     localeDetection: false,
   },
-
+  react: { useSuspense: false },
```

Comment:
What is the reason for this configuration? If the reason is not commented, later maintainers will not dare to modify here.

Reply:
It was introduced by fixing an issue of i18n, referred from https://stackoverflow.com/questions/67894982/react-i18next-you-will-need-to-pass-in-an-i18next-instance-by-using-initreacti

The issue is weird and fixed by changing nothing. This line is removed by https://github.com/Magickbase/nervos-official-website/pull/15/commits/0afbb912655e383d8d4070eef3f5c76a2471966e